test_scripts/Defects/5_0/2670_4_GetInteriorVD_after_ignition_off.lua

```diff
@@ -78,7 +77,7 @@ runner.Step("GetInteriorVehicleData RADIO", commonRC.rpcDenied,
   { "RADIO", 1, "GetInteriorVehicleData", "DISALLOWED" })
 runner.Step("ignitionOff", ignitionOff)
 runner.Step("Start SDL, HMI, connect Mobile, start Session", actions.start)
-runner.Step("RAI", actions.registerApp)
+runner.Step("RAI", actions.registerAppWOPTU)
```

Comment:
Is there a reason that a PTU is not started the first time this app is registered? It seems like it would meet the "new application" trigger.

Reply:
@jacobkeeler That's right PTU should be started. However it will be a new ignition cycle trigger (not a new application).
Actually this application was registered in a previous ignition cycle and PTU was not finished, so SDL should re-start PTU in a new ignition cycle. One additional point here - presence of application is not required due to HMI PTU feature. So PTU will be started once HMI initialized and before app re-registration.